espressif/idf-eclipse-plugin
# IEP-54 New component wizard

Allow creating the boilerplate of a component from "New..." menu or from the context menu in Project Explorer.
The new component is stored in the "components" project's folder. 
![IEP-54](https://user-images.githubusercontent.com/24419842/111834669-3951ec00-88f4-11eb-8fac-fde0c879be34.png)

Test cases:
- Select project from the Project Explorer -> Go to new -> Espressif IDF Component -> Choose name "test" -> finish button -> component is created with following information window:
![IEP-54 informal window](https://user-images.githubusercontent.com/24419842/111835239-f9d7cf80-88f4-11eb-90d3-62c70673d575.png)

- check validation name is working correctly: empty name, a name with space or component with such name is already exists - passed
